components/script/dom/serviceworker.rs

```diff
@@ -65,16 +67,6 @@ impl ServiceWorker {
     pub fn get_script_url(&self) -> Url {
         Url::parse(&self.script_url.borrow().clone()).unwrap()
     }
-
-    pub fn install_serviceworker(global: GlobalRef,
-                                 script_url: Url,
-                                 scope_url: &str,
-                                 skip_waiting: bool) -> Root<ServiceWorker> {
-        ServiceWorker::new(global,
-                           script_url.as_str(),
-                           scope_url,
-                           skip_waiting)
-    }
```

Comment:
Refactored this into a single method as this was just an unneccessary indirection.